### DISCUSSION TITLE
Option analog to `math-style` in `unicode-math`

### DISCUSSION THREAD

**Discussion by omasanori** [olsak/OpTeX#113] [Q&A] [answered]
I've been trying Euler Math (OpenType version of AMS Euler, successor of Neo Euler) with OpTeX. A problem is, Euler typefaces have no italic glyphs. Variables, displayed in italic with typical math typefaces, are in upright with Euler typefaces.
`unicode-math` has the `math-style` option to manage which kind of glyphs should be italic or not, along with other knobs. (See section 5 in `unicode-math` user manual.) While I am unsure if we want all the options `unicode-math` provides in OpTeX, they may be worth a consideration imho.

**Comment by omasanori** [comment]
Well, I overlooked section 1.9 in the OpTeX math manual...

**Comment by olsak** [comment, marked as answer]
I added these options right now. See https://github.com/olsak/OpTeX/commit/8bf17206153c670b1b3db607fadda48896ee39d6.

**Reply by omasanori** [reply]
Thank you so much!
